backend/src/main/resources/application.properties

```diff
@@ -1 +1,2 @@
-
+spring.data.mongodb.uri=mongodb+srv://dhiren:2iW2rl07hPh1wSwB@cluster0-f8bs1.azure.mongodb.net/test?retryWrites=true&w=majority
+spring.data.mongodb.database=ewp_dhiren_develop
```

Comment:
- [ ] Create a separate properties file for database related things because each developer will have their own db properties. Once you create that properties file add it to this PR. Once your PR is meged add that new file to gitignore. Also, add things to app.properties only if is going to stay common to everyone. Otherwise create a separate file. Lmk if we need to discuss.